twoliter/src/lock.rs

```diff
@@ -139,6 +170,111 @@ impl Lock {
         Ok(lock)
     }
 
+    /// Fetches all external kits defined in a Twoliter.lock to the build directory
+    pub(crate) async fn fetch(&self, project: &Project, arch: &str) -> Result<()> {
+        let target_dir = project.project_dir().join("build/external-kits");
+        create_dir_all(&target_dir)
+            .await
+            .context("failed to create external-kits directory")?;
+        let mut kits: Vec<Image> = Vec::new();
+        for image in self.kit.iter() {
+            kits.push(self.extract_kit(&target_dir, image, arch).await?);
+        }
+        let kit_list = serde_json::to_vec(&ExternalKitMetadata {
+            sdk: Image {
```

Comment:
since we do basically the same conversion in `extract_kit`, might just make sense to impl `Into<Image>` for `LockedImage`